packages/sakuli-legacy/src/context/common/actions/converter.function.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { createRegionClass } from "../region";
+import { Region as NutRegion } from "@nut-tree/nut-js/dist/lib/region.class";
+import { Project, TestExecutionContext } from "@sakuli/core";
+import { createTestExecutionContextMock } from "../../__mocks__";
+import { toNutRegion } from "./converter.function";
+
+let ctx: TestExecutionContext;
+let project: Project;
+
+beforeEach(() => {
+  ctx = createTestExecutionContextMock();
+  project = new Project("");
+});
+describe("toNutRegion", () => {
```

Comment:
I'd suggest to add a test for the "default conversion" in case region getters return `undefined`. IMHO the default values should be protected against an intended or unintended change, as a change to this values might change the behavior of everything relying on that conversion.

Reply:
Totally agree 👍 Will add an additional test